app/main.py

```diff
@@ -1,2 +1,66 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.capacity = 8
+        self.hash_table = [None for _ in range(self.capacity)]
+        self.size = 0
+
+    def __getitem__(self, item):
+        index = hash(item) % self.capacity
+
+        for _ in range(2):
+            for i in range(self.capacity):
+                if index + i == self.capacity:
```

Comment:
What about `index + i > self.capacity` case?

Reply:
when I do so it does not work (